build.gradle

```diff
@@ -18,5 +18,7 @@ checkstyleMain {
 
 task runLangtonAnt (dependsOn: 'check', type: JavaExec) {
     main = 'org.fundacionjala.at15.katas.langtonant.Main'
+    main = 'org.fundacionjala.at15.katas.langtonant.jimy.Main'
```

Comment:
You need to create new task for this. Copy `runLangtonAnt` and modify `main` to be the way you have it in this line. Also, use another name for the new task